server/routes/api/recommendations/recommendationsRoutes.ts

```diff
@@ -1,11 +1,94 @@
-import { Router } from "express";
+import { Router, Request, Response } from "express";
+import { validationResult, query, param } from "express-validator";
+
 import Handler from "./recommendationsHandler";
+
 const recommendationsRouter = Router();
 
 const handler = new Handler();
+const babysitterInvalidInputError = "babysitterId must be provided and a number";
```

Comment:
this is a const so if you want to put it outside from the function it will be a global const and you need to fit the format.
It needs to be - BABYSITTER_INVALID_INPUT_ERROR = 'babysitterId must be provided and a number';